simulariumio/tests/converters/test_cellpack_converter.py

```diff
@@ -27,6 +29,8 @@
 
 converter = CellpackConverter(data)
 results = JsonWriter.format_trajectory_data(converter._data)
+range = 200.0
+auto_scale_factor = VIEWER_DIMENSION_RANGE.MAX / range
```

Comment:
what is the auto scale factor?

Reply:
that's the scale factor that's automatically generated. I didn't spend much time thinking up a name for it since it's just in a test, but I'm open to suggestion